frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx

```diff
@@ -152,11 +153,14 @@ export default function MobileView(props: CodingViewProps) {
             <ViewTribe {...props} />
             <AddToFavorites {...props} />
             <CopyLink {...props} />
-            <ShareOnTwitter {...props} />
+            <ShareOnTwitter
+              issueCreated={created}
+              ownerPubkey={owner_id}
+              labels={labels}
+              titleString={titleString}
+            />
           </ButtonRow>
 
-          {markUnpaid}
```

Comment:
why was this removed? not sure what it does either

Reply:
oh well if you see the current deployed version in mobile view it has this mark as unpaid button. but this is not present in the desktop view. I tried to copy the desktop view and thus to get the uniformity i have removed this

Reply:
example 

<img width="494" alt="image" src="https://github.com/stakwork/sphinx-tribes/assets/91251535/16a9de6f-a6c8-4b91-88f8-4f10e4c53d33">

Reply:
The mark unpaid also exists on desktop

We just should only show the mark unpaid button when the bounty is paid

Reply:
Okay. Sorry for removing it. I'll fix it right away

Reply:
I would add `{ !!paid ? markUnpaid : null }` So it only shows when paid